### ISSUE TITLE
taking yellorium out of reactor deletes yellorium

### ISSUE THREAD

**Issue by Mummrah** [SleepingTea98/Colony-Modpack#18]
Hey

because people can acces big reactor ports and controllers what leads to yellorium being taken by other players, i was thinking of taking out the yellorium out of my reactor when going offline . i have a very big big reactor and it takes a lot of yellorium it has a max capacity of 37048000 mb , i had it 33% filled. when i took out everything (it dumped straigth into a chest) i lost almost all of it i only got 1.5 stacks of yellorium back what is not near 33% fuel ... what do i do because i need the power from my reactor and its not cheap and easy to get that back up to 33% filled... 
so long story short: removing fuel by letting it straight into a chest removes most of the yellorium

**Comment by SleepingTea98** [comment]
Closed due to modpack updating to 1.12.2
